safechef/safechef-v1.context.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": {
+    "@protected": true,
+
+    "FoodSafetyCertificationCredential": {
+      "@id": "https://credential-handler.github.io/vc-examples/safechef/#FoodSafetyCertificationCredential"
+    },
+
+    "name": "http://schema.org/name",
+    "description": "http://schema.org/description",
+    "image": "http://schema.org/image",
```

Comment:
When are we switching to the https versions of these?

Reply:
Oh. Good question. Looks like they've made the transition in their examples/docs now, so I will too. Good call.